src/Symfony/Bridge/Doctrine/Tests/Validator/Constraints/UniqueEntityValidatorTest.php

```diff
@@ -114,7 +114,7 @@ protected function createEntityManagerMock($repositoryMock)
             ->method('hasField')
             ->willReturn(true)
         ;
-        $refl = $this->createStub(\ReflectionProperty::class);
+        $refl = $this->createMock(\ReflectionProperty::class);
         $refl
```

Comment:
don't we need to keep the `->expects($this->any())` then when using `createMock` ?

Reply:
`$mock->method('foo')` is a shortcut for `$mock->expects($this->any())->method('foo')`.

Reply:
Yeah AFAIK it's a NOOP